CONTRIBUTING.md

```diff
@@ -0,0 +1,48 @@
+# 📚 Contributing
+- 🛠 [Opening an Issue](#issues)
+- 🔀 [Pull requests](#Pull-requests)
+
+
+# 🛠 Issues
+### 📝 Creating new issues
+
+🎉 Any issue, whether it be bugs, feature requests, or general feedback is welcome!
+
+Please do a quick search of past issues before opening a ticket. If you are working on a ticket, please assign yourself to it, and unassign yourself if you decide to stop working on it. Make sure your tickets are tagged with the correct labels.
+
+If you would like to make a suggestion, please create an issue with the following information included:
+- Description of the 'problem'
+- Brief justification of a solution
+
+### ⚠ Bug reporting
+
+Bugs can be reported by any one at any time. Check `bugs` label for duplicate reports.  Bug reports should contain:
+- a **description** of the bug (*what* happened)
+- detailed **steps** to reproduce the bug (*when* the bug happened)
+- a **screen shot** of the bug (if possible)
+- browser, OS, version # of Node (this can help us understand *why* the bug occurred)
+- any other **notes** (e.g. if the bug only occurs in certain situations)
+
+If you find an existing bug report that is missing information (such as if a bug can be reproduced in other ways), feel free to add details.
+
+<!-- #### Example bug report -->
+<!-- - TODO: Add an example bug report -->
```

Comment:
you can just document this as an Issue Template:

https://github.com/ubclaunchpad/inertia/issues/new?assignees=&labels=type%3A+bug+%3Abug%3A&template=bug-report.md&title=

https://docs.ubclaunchpad.com/handbook/project-management/repositories#repository-templates